.github/workflows/daily-installation-test.yml

```diff
@@ -29,12 +14,12 @@ jobs:
     strategy:
         matrix:
             go-version: ${{ fromJson(needs.go-versions.outputs.matrix) }}
+            relay-major: ['v7', 'v8']
+        fail-fast: false
     steps:
       - uses: actions/checkout@v4
-        with:
-          ref: ${{ inputs.branch }}
       - uses: ./.github/actions/install-relay
         with:
           go-version: ${{ matrix.go-version }}
-          relay-version: ${{ inputs.relay-version }}
-          relay-major: ${{ inputs.relay-major }}
+          relay-version: 'latest'
```

Comment:
I am confused by the versions.

Reply:
It's confusing. The `v7` or `v8` part is part of the module path, and then you indicate the actual tag. 

So for ex relay's latest release is:
-  `/ld-relay/v8@latest`, or
- `/ld-relay/v8@v8.3.1`

`v7`'s latest release is:
- `/ld-relay/v7@latest` or,
- `/ld-relay/v7@v7.5.0`

So it's really indicating "give me the v7 module (which is totally independent of the v8 module), and then give me a specific tag of that module."

I could perhaps have some better words to use in the yml, but I wasn't sure.